pyanaconda/ui/tui/spokes/software_selection.py

```diff
@@ -93,6 +100,11 @@ def _initialize(self):
         """Initialize the spoke in a separate thread."""
         threadMgr.wait(THREAD_PAYLOAD)
 
+        self.available_kernels = get_display_kernel_features(self.payload)
```

Comment:
In general, it is good to keep the implementation of GUI and TUI as close and consistent as possible, because usually we change both at the same time. Here TUI uses `available_kernels`, but GUI uses `show_kernel_features` for the same data. Ideally, these attributes should have the same name.